### DISCUSSION TITLE
Infinite processing on submission of evidence for Student verification

### DISCUSSION THREAD

**Discussion by HarrisonWilde** [community/community#101439] [GitHub Education]
### Hi everyone,

I have been having an issue with re-verifying my academic status, I am trying to upload a jpeg of my student status letter (lost my ID card and not currently able to get a new one). I know it says processing can take a while but I have left it for over an hour and had no luck in the application being submitted.

**Comment by meyting** [comment]
Same here (but for teacher application)!
